make-dist.sh

```diff
@@ -71,6 +71,6 @@ cp $BASEDIR/scripts/classes.lst $BIN_DIR/
 cp $BASEDIR/scripts/img_class.lst $BIN_DIR/
 cp $BASEDIR/spark/dl/src/main/resources/spark-bigdl.conf $CONF_DIR/
 
-cp $BASEDIR/spark/dl/target/bigdl-$VERSION-jar-with-dependencies.jar $LIB_DIR/
```

Comment:
On master, it's a SNAPSHOT build. On release branch(e.g. branch-0.1), it's not SNAPSHOT build.